desktop-widgets/divelistview.cpp

```diff
@@ -26,7 +26,8 @@
 #include "core/metrics.h"
 
 DiveListView::DiveListView(QWidget *parent) : QTreeView(parent), mouseClickSelection(false), sortColumn(0),
-	currentOrder(Qt::DescendingOrder), dontEmitDiveChangedSignal(false), selectionSaved(false)
+	currentOrder(Qt::DescendingOrder), dontEmitDiveChangedSignal(false), selectionSaved(false),
+	initialColumnWidths(DiveTripModel::COLUMNS, 50)		// Set up with default length 50
```

Comment:
comment should be with a single space after the code line.
missing `;` at the end.

Reply:
> comment should be with a single space after the code line.

Ok - will do

> missing ; at the end.

This is an initialization list. I think adding ';' would not compile..